frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx

```diff
@@ -131,7 +133,7 @@ export const IconsComponent: FC<Props> = ({
           role === 'moderator' ||
           role === 'admin') && (
           <MenuItem onClick={handleClose}>
-            <ListItemIcon onClick={() => null}>
+            <ListItemIcon onClick={handleDeleteClick}>
```

Comment:
<MenuItem onClick={editData}>
            <ListItemIcon>
              <EditIcon fontSize="small" />
            </ListItemIcon>
            <ListItemText>{t('createLocation.editLocation')}</ListItemText>
          </MenuItem>
          
          you should use this example for you code - because delete logic work only on icon